test/conformance/testing/include/uur/fixtures.h

```diff
@@ -1068,6 +1068,14 @@ std::string deviceTestWithParamPrinter<BoolTestParam>(
     const ::testing::TestParamInfo<
         std::tuple<ur_device_handle_t, BoolTestParam>> &info);
 
+using SamplerCreateParamTy =
```

Comment:
Is the `Ty` at the end of this name a typo?

Reply:
It should reflect an alias for a tuple that holds the sampler creation parameters. So, I guess `Ty` should reflect that it is like an alias type. But I could change that if you think that shouldn't be accurate.

Reply:
Ah is this a naming convention? Never came across it before.

Reply:
`Ty` is usually form `llvm::Type*` variables, though. I don't think its UR naming convention for an actual type. That would either be `T` in PascalCase names or `_t` in snake_case names.

Reply:
I changed it to a `T`. I didn't notice that `Ty` is not common in UR codebase.